omero/users/cli/import-bulk.rst

```diff
@@ -0,0 +1,99 @@
+Using bulk imports
+==================
+
+The CLI import option ``--bulk`` specifies a configuration file that
+can be used to perform a batch of imports with the same or similar
+options. The file is written in a simple YAML syntax and can be placed
+anywhere.
+
+A simple YAML file might look like: ::
+
+    ---
+    continue: "true"
+    transfer: "ln_s"
+    path: "my-files.txt"
```

Comment:
Yeah, I wasn't sure if we wanted to perhaps conclude with an IDR example.

Reply:
Outreach in Basel has shown that the idr examples are far too complex (and even possibly not the most useful for OMERO users without simplifications). 

Also, we were explicitly asked for an example or possiblity to pass a whole folder into the ``tsv`` file instead of single files. 
We tested that on the spot and it worked.
I do not see such example in the whole doc here cc @jburel

Reply:
I will add a directory example, but similar to @sbesson's statement, I would prefer to *not* try to get all of your knowledge/examples from training into this PR. If someone would like to open a follow-up, feel free.